From 1e4ce976172e3bf46f3dc6f170dce8fa8f412b6f Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Tue, 26 Jan 2016 09:59:29 -0600
Subject: [PATCH 01/10] Ceph sudoers entry

---
 ceph.sudoers | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 ceph.sudoers

diff --git a/ceph.sudoers b/ceph.sudoers
new file mode 100644
index 000000000..a1ff75a2a
--- /dev/null
+++ b/ceph.sudoers
@@ -0,0 +1 @@
+ceph ALL = (root) NOPASSWD: ALL
-- 
2.38.1

