From efd82486376727f18fe473a2da22a3b54f16580a Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Tue, 26 Jan 2016 09:59:29 -0600
Subject: [PATCH 1/2] Ceph sudoers entry

---
 ceph.sudoers | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 ceph.sudoers

diff --git a/ceph.sudoers b/ceph.sudoers
new file mode 100644
index 0000000..a1ff75a
--- /dev/null
+++ b/ceph.sudoers
@@ -0,0 +1 @@
+ceph ALL = (root) NOPASSWD: ALL
-- 
2.5.0

