From f3193361b412211c7875d6fe699e0dd102907114 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Tue, 26 Jan 2016 09:59:29 -0600
Subject: [PATCH 1/8] Ceph sudoers entry

---
 ceph.sudoers | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 ceph.sudoers

diff --git a/ceph.sudoers b/ceph.sudoers
new file mode 100644
index 00000000..a1ff75a2
--- /dev/null
+++ b/ceph.sudoers
@@ -0,0 +1 @@
+ceph ALL = (root) NOPASSWD: ALL
-- 
2.13.1

