From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Thu, 9 Mar 2017 17:59:44 -0600
Subject: [PATCH] bash-completion

---
 src/bash_completion/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bash_completion/CMakeLists.txt b/src/bash_completion/CMakeLists.txt
index 8d834774..7c27c277 100644
--- a/src/bash_completion/CMakeLists.txt
+++ b/src/bash_completion/CMakeLists.txt
@@ -11,5 +11,5 @@ if(WITH_RADOSGW)
 endif()
 
 install(FILES ${completions}
-  DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/bash_completion.d)
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/share/bash-completion/completions)
 
