From 9eca2dc708fe1cd5d6b0b5403363cd3a7ac580f5 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Tue, 21 Jun 2016 10:50:38 -0500
Subject: [PATCH 4/7] use uname instead of arch

---
 configure    | 2 +-
 configure.ac | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index d434fb9..3cf7456 100755
--- a/configure
+++ b/configure
@@ -18470,7 +18470,7 @@ fi
 if test x"$YASM_CHECK" = x"yes"; then
   if yasm -f elf64 src/common/crc32c_intel_fast_asm.S -o /dev/null; then
      echo 'we have a modern and working yasm'
-     if test `arch` = "x86_64" ; then
+     if test `uname -m` = "x86_64" ; then
         echo 'we are x86_64'
         arch_x32=0
         cat confdefs.h - <<_ACEOF >conftest.$ac_ext
diff --git a/configure.ac b/configure.ac
index 32e273f..28883cd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -244,7 +244,7 @@ AC_CHECK_PROG(YASM_CHECK, yasm, yes)
 if test x"$YASM_CHECK" = x"yes"; then
   if yasm -f elf64 src/common/crc32c_intel_fast_asm.S -o /dev/null; then
      echo 'we have a modern and working yasm'
-     if test `arch` = "x86_64" ; then
+     if test `uname -m` = "x86_64" ; then
         echo 'we are x86_64'
         arch_x32=0
         AC_COMPILE_IFELSE([AC_LANG_PROGRAM([], [[
-- 
2.7.4

