From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Thu, 9 Mar 2017 18:03:41 -0600
Subject: [PATCH] os-release

---
 src/common/util.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/common/util.cc b/src/common/util.cc
index f816ff41..60707f07 100644
--- a/src/common/util.cc
+++ b/src/common/util.cc
@@ -101,10 +101,10 @@ static bool os_release_parse(map<string, string> *m, CephContext *cct)
     { "distro_version", "VERSION_ID=" }
   };
 
-  FILE *fp = fopen("/etc/os-release", "r");
+  FILE *fp = fopen("/usr/lib/os-release", "r");
   if (!fp) {
     int ret = -errno;
-    lderr(cct) << "os_release_parse - failed to open /etc/os-release: " << cpp_strerror(ret) << dendl;
+    lderr(cct) << "os_release_parse - failed to open /usr/lib/os-release: " << cpp_strerror(ret) << dendl;
     return false;
   }
 
@@ -127,7 +127,7 @@ static bool os_release_parse(map<string, string> *m, CephContext *cct)
 static void distro_detect(map<string, string> *m, CephContext *cct)
 {
   if (!os_release_parse(m, cct)) {
-    lderr(cct) << "distro_detect - /etc/os-release is required" << dendl;
+    lderr(cct) << "distro_detect - /usr/lib/os-release is required" << dendl;
   }
 
   for (const char* rk: {"distro", "distro_description"}) {
