From 5c3a81b59478d5dda07d83db9e9ea098c6bce16d Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 15 Nov 2021 14:24:20 -0800
Subject: [PATCH] Fix missing include

---
 src/rgw/rgw_string.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/rgw/rgw_string.h b/src/rgw/rgw_string.h
index 257daa9c1..90e64f98a 100644
--- a/src/rgw/rgw_string.h
+++ b/src/rgw/rgw_string.h
@@ -8,6 +8,8 @@
 #include <stdlib.h>
 #include <limits.h>
 #include <string_view>
+#include <string>
+#include <stdexcept>
 
 #include <boost/container/small_vector.hpp>
 
-- 
2.33.0

