--- ceph-13.2.0/src/mgr/PythonCompat.h~	2018-05-29 12:04:02.000000000 +0000
+++ ceph-13.2.0/src/mgr/PythonCompat.h	2018-06-30 21:46:50.643532726 +0000
@@ -13,7 +13,7 @@
 inline PyObject* PyString_FromString(const char *v) {
   return PyUnicode_FromFormat("%s", v);
 }
-inline char* PyString_AsString(PyObject *string) {
+inline const char* PyString_AsString(PyObject *string) {
   return PyUnicode_AsUTF8(string);
 }
 inline long PyInt_AsLong(PyObject *io) {
--- ceph-13.2.0/src/boost/boost/python/detail/caller.hpp~	2018-07-04 13:06:25.000000000 +0000
+++ ceph-13.2.0/src/boost/boost/python/detail/caller.hpp	2018-07-04 15:15:39.423298389 +0000
@@ -50,7 +50,7 @@
 
 inline Py_ssize_t arity(PyObject* const& args_)
 {
-    return PyTuple_GET_SIZE(args_);
+    return Py_SIZE(args_);
 }
 
 // This "result converter" is really just used as
