--- ceph-12.0.3/src/rocksdb/CMakeLists.txt~	2017-04-27 00:13:46.000000000 +0000
+++ ceph-12.0.3/src/rocksdb/CMakeLists.txt	2017-06-07 15:44:57.080395312 +0000
@@ -161,7 +161,7 @@
   if(MSVC)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /WX")
   else() # assume GCC
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
   endif()
 endif()
 
--- ceph-12.0.3/src/dmclock/CMakeLists.txt~	2017-05-16 12:42:56.000000000 +0000
+++ ceph-12.0.3/src/dmclock/CMakeLists.txt	2017-06-07 17:49:14.065345952 +0000
@@ -17,7 +17,7 @@
 endif()
 
 if (NOT(TARGET gtest AND TARGET gtest_main))
-  find_package(gtest REQUIRED)
+  find_package(GTest REQUIRED)
   include_directories(${GTEST_INCLUDE_DIRS})
 endif()
 
